spa-checklist.md

```diff
@@ -30,20 +30,34 @@ This is a checklist for browser-based Web App **without the backend** or with a
 
 ## Versioning
 
- - [ ] My app has a cache-busting implemented 
+ - [ ] My app has a assets cache-busting implemented
```

Comment:
Please, remove an article in the sentence: 
"My app has assets cache-busting implemented"